src/auto_code_surveys.py

```diff
@@ -51,8 +70,8 @@ def auto_code_surveys(cls, user, data, icr_output_dir, coda_output_dir):
             icr_output_path = path.join(icr_output_dir, plan.icr_filename)
             with open(icr_output_path, "w") as f:
                 TracedDataCSVIO.export_traced_data_iterable_to_csv(
-                    icr_messages, f, headers=[plan.run_id_field, plan.raw_field]
+                    icr_messages, f, headers=[plan.raw_field]
                 )
 
 
-        return data
+        return data
```

Comment:
new line

Reply:
Added

Reply:
And set (again?) VSCode to do it automatically